tests/ha/ha_cluster_crash_test.pm

```diff
@@ -68,12 +71,28 @@ sub run {
             }
             sleep 4;
         }
+
+        if ($check eq "kill-pacemakerd") {
+            if ($node_was_fenced == 1) {
```

Comment:
```suggestion
            if ($node_was_fenced) {
```

Reply:
There are other instances in the code where we have this redundant boolean check.

Reply:
Thanks, I have changed all of them.